### ISSUE TITLE
Feature Request: Switch pre-fader on the mixer

### ISSUE THREAD

**Issue by soulbridge** [mschack/VCV-Rack-Plugins#6]
Hi, it would be useful to have a switch on the red outputs to set them on pre-fader mode in order to chain effects that hasn't dry/wet control, what do you think?
![cattura di schermata 58](https://user-images.githubusercontent.com/32793048/32492203-cf5d6282-c3b9-11e7-9efa-5e49dfd090b1.png)

**Comment by soulbridge** [comment]
sorry I didn't notice that the purples in are inputs cv for mute and solo hahah however the idea it's always the same if u like it

**Comment by mschack** [comment]
Yeah I had the same idea.  Amazing the simple things you come up with when you actually start trying to use this stuff...  Or I may sacrifice the mute/solo inputs for separate pre-fader outputs, as they aren't as useful I think.

**Comment by soulbridge** [comment]
probably they're not very useful but also not completely useless, i suppose it depends on the situation...
however personally i prefer a send and return than a cv for mute and solo

**Comment by mschack** [comment]
Trying to incorporate 4 aux channel faders per group (A, B etc.) each with a pre fader select button, then there will be four aux outs along the bottom.  Might only be two due to space restrictions, but it might work

**Comment by mschack** [comment]
Finished.  Give it a try and tell me what you think.

**Comment by soulbridge** [comment]
I'm home now. Good... let me see it.

**Comment by mschack** [comment]
beware, all the level faders can now amplify to 4x so it can really get loud.

**Comment by soulbridge** [comment]
I noticed. I like the idea to use 4 aux channels however prefaders doesn't work I don't know why and also the inputs of levels and pans of all blocks, they are cv controls or audio inputs? 
by the way prefaders on the aux channels are useless cause

before in the first example, supposed i don't have an effect with dry/wet knob, i need the switches for prefader to block the signal A towards B in order to have:

 A sends to Effects in prefader, the knob of A towards B it's closed (dry signal), then effected signal returns to B

in this case a switch for prefader on the aux won't be useful cause i can close the A to B knob while i send the signal on the aux with no problem. 

In any case good work, you can remove the switches and fix the inputs (audio or cv) near master level and master pan of blocks.

**Comment by mschack** [comment]
Levels and pan are CV.  The pre fader works for me, I suppose the one use would be to get the output to the aux channel independent of mute or level.  I am currently making 1x4 and 2x4 mixers, I will look into these issues when I test them.  

Try also to initialize the mixer.  Since I added new params they may have conflicted with your old mixer settings.

Thanks for trying this out.

**Comment by soulbridge** [comment]
I didn't understand pre-faders referred to level inputs, ok now i figured it out. I tried again the cv inputs the pans works but levels cv not.

**Comment by mschack** [comment]
whar are cv levels? 0 - 10?

**Comment by soulbridge** [comment]
I tried from -10 to +10 but nothing happens to be sure i also tried from -100 to 100.

**Comment by mschack** [comment]
ok, thanks

**Comment by mschack** [comment]
Fixed a bunch of stuff, should work better now.

**Comment by soulbridge** [comment]
mixer 4*4 seems to work perfectly 
mixer 1*4 and 2*4 have an issue on channel level knobs cause cause i can hear just the aux outs with prefader switches on.
